include/usb/class/usb_cdc.h

```diff
@@ -65,6 +65,8 @@
  * for ACM devices
  * @note PSTN120.pdf, 6.3, Table 13
  */
+#define CDC_SEND_ENC_CMD		0x00
+#define CDC_GET_ENC_RSP			0x01
```

Comment:
Is github messing up the order of the commits again? I guess this commit comes _before_ adding RNDIS

Reply:
Yes, it comes before.